Dockerfile

```diff
@@ -15,11 +15,11 @@ ENV RALPH_STATIC=/root/static
 ENV RALPH_DOCS=$RALPH_DIR/docs
 ENV SCRIPTS_PATH=/root
 
-ADD docker/* $SCRIPTS_PATH/
-
 # basic provisioning
 RUN $SCRIPTS_PATH/provision.sh
 
+ADD docker/* $SCRIPTS_PATH/
```

Comment:
this will not work - first you run provision script, then you add this script. How should it work? Run `provision.sh` is cached until someone change something  in `docker/` directory.